### ISSUE TITLE
java.io.NotSerializableException: scalapb.TypeMapper$$anon$1

### ISSUE THREAD

**Issue by girishpandit88** [scalapb/ScalaPB#884]
I'm trying to deserialize a `blob` Cassandra table column that is a protobuf. 

```
Caused by: java.io.NotSerializableException: scalapb.TypeMapper$$anon$1
Serialization stack:
	- object not serializable (class: scalapb.TypeMapper$$anon$1, value: scalapb.TypeMapper$$anon$1@1835b24b)
	- field (class: foo.bar.Context$DataEntry$, name: keyValueMapper, type: class scalapb.TypeMapper)
	- object (class foo.bar.Context$DataEntry$, foo.bar.Context$DataEntry$@42f15e87)
```

proto message causing the issue:

```
message ContextValue {
    oneof value {
        string string_value = 1;
        double double_value = 2;
        int64 long_value = 3;
    }
}

message Msg {
...
Context context = 1;
}

message Context {
    map<string, ContextValue> data = 1;
}
```
scala code that tries to parse the protobuf binary blob and append the deserialized column
```scala
import scalapb.spark.Implicits._
import scalapb.spark.ProtoSQL
import sparkSession.implicits.StringToColumn
val parseData = ProtoSQL.udf { bytes: Array[Byte] => Msg.parseFrom(bytes) }
df.withColumn("parsed", parseData($"data")) //data is the column that points to the protobuf binary blob.
```

build.gradle:

```
dependencies {
    compile "com.thesamet.scalapb:scalapb-runtime_2.11:0.9.+"
    compile "com.thesamet.scalapb:sparksql-scalapb_2.11:0.9.1"
    compile "org.apache.spark:spark-core_2.11:2.4.6"
    compile "org.apache.spark:spark-sql_2.11:2.4.6"
}
```

using jdk11. 

is `map` causing the serialization? This is an upstream proto so I cannot modify it.

**Comment by thesamet** [comment]
@girishpandit88 This looks like #841 which has been resolved for 0.10.x. I backported the fix to 0.9.x and released 0.9.3. Let me know if this works for you.

**Comment by girishpandit88** [comment]
yes that works. Thanks for the pointers. You can close this issue.
